regress/core/dump.sql

```diff
@@ -178,3 +178,14 @@ GEOMETRYCOLLECTION (
 	)
 )
 ');
+
+-- Check that it works without the extension schema being available
+SET search_path TO pg_catalog;
+WITH data AS
+(
+    SELECT 'POLYGON((0 0, 0 1, 1 1, 1 0, 0 0))'::public.geometry as geom
```

Comment:
I don't fully like the assumption that PostGIS is installed in the "public" schema.
Especially since we were talking about adding tests for it being installed in a foreign schema.
Any chance to use psql variables to define a schema ?

Reply:
Seems to work:
```
strk=# \set schema public.
strk=# select 'POINT(0 0)':: :schema geometry;
                  geometry                  
--------------------------------------------
 010100000000000000000000000000000000000000
(1 row)
```

Reply:
I think it's doable. I could add it in the `run_simple_test` and set it to an option or `public` if not set (OPT_POSTGIS_SCHEMA or something like that).